.github/PULL_REQUEST_TEMPLATE/rosdep_pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+<!-- Submitter Please review the contributing guidelines: https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md -->
+
+Please add the following dependency to the rosdep database.
+
+## Package name:
+
+TODO
+
+## Package Upstream Source:
+
+TODO link to source repository
+
+## Purpose of using this:
+
+TODO Replace. This dependency is being used for this reason. This is why I think it's valuable to be added to the rosdep database. 
+
+Distro packaging links:
+
+## Links to Distribution Packages
+
+  Debian: https://packages.debian.org/
+    - REQUIRED
+  Ubuntu: https://packages.ubuntu.com/
+    - REQUIRED
+  Fedora: https://apps.fedoraproject.org/
+    - If AVAILABLE
+  Arch: https://www.archlinux.org/packages/
+    - IF AVAILABLE
+  Gentoo: https://packages.gentoo.org/packages/
+    - IF AVAILABLE
+  macOS: https://formulae.brew.sh/
+    - IF AVAILABLE
```

Comment:
Formatting is lost on this section when the markdown is rendered.

Reply:
Fixed.